tools/codegen/dest/lazy_ir.py

```diff
@@ -23,6 +23,8 @@ def node_ctor_arg_rvalue_string(arg: LazyArgument) -> str:
                 return f"torch::lazy::LazyGraphExecutor::Get()->GetIrValueForScalarFromCodegen({arg.name})"
             elif arg.lazy_type.type is tensorListValueT:
                 return f"lazy_{arg.name}_tensorlist"
+            elif arg.is_symint_or_list:
```

Comment:
does 'or_list' mean list of symint in this case?

is this generated code correct for both cases? (maybe, as it is a base class)

Reply:
yes, I'm thinking of reusing this flag for both symints and list of symints, we don't quite yet have lists of symints if it's confusing please let me know I'll rename the flag latter.